python/paddle/v2/dataset/common.py

```diff
@@ -35,15 +36,30 @@ def md5file(fname):
 
 
 def download(url, module_name, md5sum):
+    print "downloading %s" % url
```

Comment:
👍！
Done. Changed to `print "Cache file %s not found, downloading %s" % (filename, url)` (`cache file` instead of `cache dir`)